.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+
+services:
+  - docker
+
+language:
+  - ruby
+
+env:
+  - DOCKER_COMPOSE_VERSION=1.22.0
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
```

Comment:
This is better than what I was doing ... somehow I missed these builds!

Reply:
Same comment about short switches here

curl

`-L` => `--location`

uname

`-s` => `--kernel-name`
`-m` => `--machine`